src/components/structures/LoggedInView.tsx

```diff
@@ -444,6 +445,18 @@ class LoggedInView extends React.Component<IProps, IState> {
                 }
                 break;
 
+            case Key.H:
+                if (ev.altKey && modKey) {
+                    dis.dispatch({
+                        action: 'view_home_page',
+                    });
+                    if (window.singletonModalManager.hasDialogs()) {
+                        window.singletonModalManager.getCurrentModal().close();
+                    }
```

Comment:
this should get imported rather than accessing it directly off the window

Reply:
The ci also reminded me that the method `getCurrentModal` is private, so this approach can not work. Is it okay to make this method public (I do not see any other way to easily close the currently open modal)?

Reply:
You could add a method to the ModalManager called `closeCurrentModal` to make the intent clear

Reply:
I fixed it now in the last commit (and added the sign-off, which is why CI needs to run again)

Reply:
In future avoid force pushing as it discards my review and means I have to review from scratch instead of just the delta. Thanks

Reply:
Sorry for that and all the work I have caused with this. Thanks for still reviewing it though :)